src/game/system.h

```diff
@@ -125,6 +127,24 @@ void system_mouse_get_relative_state(int *x, int *y);
  */
 void system_move_mouse_cursor(int delta_x, int delta_y);
 
+/**
+ * Tells whether to use the software or hardware mouse cursor
+ * @return True when the software cursor should be used, false otherwise
+ */
+int system_use_software_cursor(void);
```

Comment:
Only use `game/system.h` for code outside `platform` that needs to call something that's implemented in `platform`, to keep the integration layer as thin as possible. These three new functions are only used inside`platform` code, so move them to a private header there.

Reply:
Well, live and learn! There are still some coding standards intrincancies I never noticed 😉